doc/manual/installation.md

```diff
@@ -53,9 +53,15 @@ Install the required packages (needed to compile Ruby and native extensions to R
     sudo apt-get install -y runit build-essential git zlib1g-dev libyaml-dev libssl-dev libgdbm-dev libreadline-dev libncurses5-dev libffi-dev curl openssh-server checkinstall libxml2-dev libxslt-dev libcurl4-openssl-dev libicu-dev logrotate python-docutils pkg-config cmake nodejs graphviz
 
 
-## 2. Ruby
+### Debian Stretch
+
+Since Debian Stretch, `runit` isn't started anymore automatically, but this gets handled by the init system. For a default installation use this package:
+
+     sudo apt-get install -y runit-systemd
```

Comment:
Can you add `libssl1.0-dev` here? Ruby 2.3.4 does not compile with the openssl 1.1 headers Stretch ships with.